[BUG]Failed to build alluxio docker image with version 2.7.0

**What is your environment(Kubernetes version, Fluid version, etc.)**

**Describe the bug**

```
bash build-image.sh
[INFO] --------------------------------------------------------------$
---------
[INFO] BUILD FAILURE
[INFO] --------------------------------------------------------------$---------
[INFO] Total time:  56.220 s
[INFO] Finished at: 2021-11-20T04:01:46Z
[INFO] --------------------------------------------------------------$---------
[WARNING] The requested profile "no-webui-linter" could not be activa$ed because it does not exist.
[ERROR] Failed to execute goal on project alluxio-underfs-hdfs: Could not resolve dependencies for project org.alluxio:alluxio-underfs-hdfs$jar:2.7.1-SNAPSHOT: Could not find artifact org.alluxio:alluxio-shade$-hadoop:jar:2.10.1 in central (https://repo1.maven.org/maven2) -> [He$p 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with $he -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions,
please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/Depe
ndencyResolutionException
[ERROR]
[ERROR] After correcting the problems, you can resume the build
```

**What you expect to happen:**

**How to reproduce it**

**Additional Information**

--- a/charts/fluid/fluid/VERSION
+++ b/charts/fluid/fluid/VERSION
@@ -71,9 +71,9 @@ release-2.5.0-2-SNAPSHOT-52ad95c
 Version: https://github.com/Alluxio/alluxio/commit/52ad95ca6b6af5c19a73a48f01a15db9d81ea38c
 Branch: https://github.com/alluxio/alluxio/tree/release-2.5.0-2
 
-20211120:
+20211123:
 
-release-2.7.0-SNAPSHOT-281696a
+release-2.7.0-SNAPSHOT-c058736
 
-Version: https://github.com/Alluxio/alluxio/commit/281696ab0c68db1aa6b520a9d8754f4bec3e88af
+Version: https://github.com/Alluxio/alluxio/commit/c05873681a824c8e056b71e3b97cb3706ed63c91
 Branch: https://github.com/Alluxio/alluxio/commits/release-2.7.0

--- a/charts/fluid/fluid/values.yaml
+++ b/charts/fluid/fluid/values.yaml
@@ -30,9 +30,9 @@ runtime:
     controller:
       image: fluidcloudnative/alluxioruntime-controller:v0.7.0-6cfba89
     runtime:
-      image: registry.aliyuncs.com/alluxio/alluxio:release-2.7.0-SNAPSHOT-281696a
+      image: registry.aliyuncs.com/alluxio/alluxio:release-2.7.0-SNAPSHOT-c058736
     fuse:
-      image: registry.aliyuncs.com/alluxio/alluxio-fuse:release-2.7.0-SNAPSHOT-281696a
+      image: registry.aliyuncs.com/alluxio/alluxio-fuse:release-2.7.0-SNAPSHOT-c058736
   jindo:
     runtimeWorkers: 3
     portRange: 18000-19999

